package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hmpps-contacts-ui",
   "version": "0.0.1",
-  "description": "Hmpps Contacts Ui",
+  "description": "Digital Prison Services",
```

Comment:
dont change the description as it should be unique to this service?

Reply:
Hmpps Contacts Ui added back